src/components/Carousel.tsx

```diff
@@ -11,8 +12,21 @@ const Carousel: React.FC<Props> = ({ images }) => {
   const [visibleImages, setVisibleImages] = useState(3);
   const [step, setStep] = useState(3);
   const [animationDuration, setAnimationDuration] = useState(1000);
+  const [inputValue, setInputValue] = useState(130);
 
-  const containerWidth = (currentIndex + visibleImages) * imageSize;
+  const wrapper = useCallback(
+    debounce((value: number) => {
+      setImageSize(value);
+    }, 1000),
+    [],
+  );
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const value = +event.target.value;
```

Comment:
better use explicit type conversion
```suggestion
const value = Number(event.target.value);

```